A60-xds-stateful-session-affinity-weighted-clusters.md

```diff
@@ -219,11 +220,10 @@ continue to use the same environment variable to also guard this feature.
 This environment variable protection will be removed once the feature has
 proven stable.
 
-These design proposes some structural or declarative changes in the source code
-which cannot be guarded by a runtime environmental variable so these will be
-present in the source code. These are:
+This design proposes some structural or declarative changes in the source code
```

Comment:
I don't think we need to list the things that are *not* guarded by the env var, only the things that *are*.  There should only be two bullets needed here.

Reply:
I don't think we need the env var to guard the following items:

- setting the call attributes for the ref-counted cluster map and route-config
- config selector will set the new call-attributes for the cluster-map, current route entry and the route config
- the new `ClusterSelectionFilter` will be added to the filter registry and added as the second-last filter in the filter chain

Those things will all happen unconditionally, because that approach supports the current functionality just fine, so it will be covered by our existing tests.

The only behavior that we need to guard with the env var are the two remaining items:

- the `StatefulSessionFilter` will add the cluster name to the cookie in the response side processing
- the `StatefulSessionFilter` will look for the cluster name in the cookie and set the cluster call-attribute in the request side processing

Reply:
done